__tests__/__integration__/extension.integration.test.ts

```diff
@@ -603,78 +603,198 @@ describe("Extension Integration Tests", () => {
     describe("Migrating a data set", () => {
         describe("Success Scenarios", () => {
             describe("Migrate a sequential data set", () => {
-                const dataSetName = `${pattern}.SDATA.SET`;
+                const dataSetName = `${pattern}.SDATA.MIGR`;
 
                 beforeEach(async () => {
-                    await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
-                    await zowe.Create.dataSet(
-                        sessionNode.getSession(),
-                        zowe.CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL,
-                        dataSetName,
-                    );
+                    await Promise.all([
+                        zowe.Delete.dataSet(
+                            sessionNode.getSession(),
+                            dataSetName
+                        ),
+                        zowe.Create.dataSet(
+                            sessionNode.getSession(),
+                            zowe.CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL,
+                            dataSetName,
+                        ),
+                    ].map((p) => p.catch((err) => err)));
```

Comment:
I think you're doing this because you want the data set to be reset but I dont like the implementation.
I have done this Promise.all trick in the past for promises where I dont care if they fail but because they are both dependent on the same resource I would run them sequentially.
A solution I would try is to await the first promise and return the error instead of throwing in case the ds doesnt exist yet
Then await the second promise but dont change the catch for this promise as if we cant create the ds we cant really run the tests